sql/moz-fx-data-shared-prod/firefox_accounts/docker_fxa_admin_server_sanitized/view.sql

```diff
@@ -2,6 +2,83 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.docker_fxa_admin_server_sanitized`
 AS
 SELECT
-  *
+  * REPLACE(
+    STRUCT(
+      resource.type,
+      STRUCT(
+        resource.labels.project_id,
+        CAST(NULL AS STRING) AS pod_name,
+        CAST(NULL AS STRING) AS container_name,
+        CAST(NULL AS STRING) AS cluster_name,
+        CAST(NULL AS STRING) AS namespace_name,
+        CAST(NULL AS STRING) AS location,
+        CAST(NULL AS STRING) AS zone,
+        CAST(NULL AS STRING) AS instance_id
```

Comment:
v1 does have `zone` and `instance_id`.
```suggestion
        resource.labels.zone,
        resource.labels.instance_id
```